### PR TITLE
feature3 added

### PR DESCRIPTION
**Pull request by aneeshjoy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ Test
 Feature1 added
 Feature2 added
 feature2 defect fix
+feature3 added
```
Comment (inbox80) [inline README.md]: add a eof line

Comment (aneeshjoy) [reply README.md]: ok. i will do it

File: README.md [hunk 2]
```diff
@@ -6,3 +6,4 @@ Feature1 added
 Feature2 added
 feature2 defect fix
 feature3 added
+eof
```
Comment (inbox80) [inline README.md]: I am happy

**Comment by aneeshjoy** [comment]
ok. i am done

**Pull request merged**